ocs_ci/ocs/const_must_gather.py

```diff
@@ -0,0 +1,162 @@
+must_gather_files = {
```

Comment:
Assumed you  got this list from actual results, you should compare it to expected results: https://github.com/openshift/ocs-operator/tree/master/must-gather/collection-scripts